github/info.json

```diff
@@ -2193,6 +2193,149 @@
         }
       ]
     },
+    {
+      "operation": "list_repository_collaborator",
+      "title": "List Repository Collaborator",
+      "description": "List the collaborator details for specified repository based on the repository owner name, repository name, and other input parameters you have specified",
+      "category": "investigation",
+      "annotation": "list_repository_collaborator",
+      "enabled": true,
+      "output_schema": [
+        {
+          "login": "",
+          "id": "",
+          "node_id": "",
+          "avatar_url": "",
+          "gravatar_id": "",
+          "url": "",
+          "html_url": "",
+          "followers_url": "",
+          "following_url": "",
+          "gists_url": "",
+          "starred_url": "",
+          "subscriptions_url": "",
+          "organizations_url": "",
+          "repos_url": "",
+          "events_url": "",
+          "received_events_url": "",
+          "type": "",
+          "site_admin": "",
+          "permissions": {
+            "admin": "",
+            "maintain": "",
+            "push": "",
+            "triage": "",
+            "pull": ""
+          },
+          "role_name": ""
+        }
+      ],
+      "parameters": [
+        {
+          "title": "Repository Type",
+          "name": "repo_type",
+          "visible": true,
+          "required": true,
+          "editable": true,
+          "type": "select",
+          "options": [
+            "Organization",
+            "User"
+          ],
+          "onchange": {
+            "Organization": [
+              {
+                "title": "Organization Name",
+                "name": "org",
+                "visible": true,
+                "required": true,
+                "editable": true,
+                "type": "text",
+                "tooltip": "Specify the name of the organization repository on GitHub in which you want to get the collaborators",
+                "description": "Specify the name of the organization repository on GitHub in which you want to get the collaborators"
+              }
+            ],
+            "User": [
+              {
+                "title": "Repository Owner",
+                "name": "owner",
+                "visible": true,
+                "required": true,
+                "editable": true,
+                "type": "text",
+                "tooltip": "Specify the account holder of the repository on GitHub in which you want to get the collaborators",
+                "description": "Specify the account holder of the repository on GitHub in which you want to get the collaborators"
```

Comment:
Suggestion: Specify the account owner of the repository. The name is not case sensitive.
Please refer to github api docs: https://docs.github.com/en/rest/collaborators/collaborators?apiVersion=2022-11-28